metrics.md

```diff
@@ -221,11 +223,11 @@ In the metrics below, we adopt the following nomenclature:
 
 ### Brier Score (BS)
 {: .anchor }
-The BS measures the accuracy of forecast probability for one or more events:
+The BS measures the accuracy of forecast probability for one or more events ([Brier50](#ref-brier50)). For single event (binary) outcomes, BS is defined as:
```

Comment:
Suggest "For events with binary outcomes," but not a strong preference.  "Events" plural because the score is calculated for a time series of random variables, where each variable indicates the (binary) outcome of a separate event.

Reply:
I like your suggested version. Revising accordingly.